packages/cli/src/commands/up.ts

```diff
@@ -106,6 +107,7 @@ export default class Up extends MachineCreationDriverCommand<typeof Up> {
       debug: flags.debug,
       machineDriver: driver,
       machineCreationDriver,
+      userModel: await this.ensureUserModel(),
```

Comment:
The (this.ensureUserModel) can extracted to a variable?
We're using it also afterwards

Reply:
done